### PR TITLE
Added locus link-out for genoox Franklin.

### PR DESCRIPTION
**Pull request by stolpeo**
Closes: #748
Related-Issue: #748
Projected-Results-Impact: none

### CODE REVIEW

**Comment by stolpeo** [comment]
@your-highness please review this as well, then I can build a new Docker image with the #829 ticket.

File: variants/templates/variants/filter_result/row.html [hunk 1]
```diff
@@ -572,6 +572,17 @@
             </span>
           {% endif %}
 
+          {% linkout_available_for "franklin" entry.release as franklin_available %}
+          {% if franklin_available %}
+            <a class="dropdown-item" href="{% linkout_base_url "franklin" entry.release %}{{ entry.chromosome }}-{{ entry.start }}-{{ entry.reference }}-{{ entry.alternative }}" target="_blank">
```
Comment (your-highness) [inline variants/templates/variants/filter_result/row.html:577]: `-{{ entry.alternative }}` should be suffixed by `-hg38` for GRCh38 or no suffix for hg19/GRCh37

File: svs/templates/svs/filter_result/row_sv.html [hunk 2]
```diff
@@ -182,6 +182,15 @@
         <a class="dropdown-item" href="http://gnomad.broadinstitute.org/region/{{ first_entry.chromosome }}-{{ first_entry.start }}-{{ first_entry.end }}" target="_blank">
           Locus @gnomAD
         </a>
+        {% if first_entry.sv_type == "DEL" or first_entry.sv_type == "DUP" %}
+          <a class="dropdown-item" href="{% linkout_base_url "franklin_sv" first_entry.release %}{{ first_entry.chromosome }}-{{ first_entry.start }}-{{ first_entry.end }}-{{ first_entry.sv_type }}" target="_blank">
```
Comment (your-highness) [inline svs/templates/svs/filter_result/row_sv.html:186]: `-{{ entry.alternative }}` should be suffixed by `-hg38` for GRCh38 or no suffix for hg19/GRCh37

**Review by your-highness — CHANGES_REQUESTED** [review]
See inline comments

**Comment by stolpeo** [comment]
@your-highness please re-review

File: variants/templates/variants/filter_result/row.html [hunk 3]
```diff
@@ -578,6 +578,17 @@
             </span>
           {% endif %}
 
+          {% linkout_available_for "franklin" entry.release as franklin_available %}
+          {% if franklin_available %}
+            <a class="dropdown-item" href="{% linkout_base_url "franklin" entry.release %}{{ entry.chromosome }}-{{ entry.start }}-{{ entry.reference }}-{{ entry.alternative }}{% if entry.release == "GRCh38" %}-hg38{% endif %}" target="_blank">
```
Comment (your-highness) [inline variants/templates/variants/filter_result/row.html:583]: Looks good!

**Review by your-highness — APPROVED** [review]
LGTM! @holtgrewe , your turn 🛩️

**Review by holtgrewe — APPROVED** [review]

**Pull request merged**